### PR TITLE
Solve some bugs

### PR DESCRIPTION
**Pull request by lldelisle**
Major:
- line_width for vlines was ignored
- in coverage when using -log was not tested correctly for potential log of negative values

Minor:
- better error handling with custom color
- more stringent on bed12 (like in UCSC)
- do not show a warning when using `file_type = x_axis`
- only plot scale bar when in the region

### CODE REVIEW

**Comment by lldelisle** [comment]
This one is green on travis on my repo: https://travis-ci.com/github/lldelisle/pyGenomeTracks/builds/175550530 but I cannot relaunch the linting step here...

File: pygenometracks/utilities.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ def transform(score_list, transform, log_pseudocount, file):
         else:
             return(np.log1p(score_list))
     elif transform == '-log':
-        if np.nanmax(score_list) <= - log_pseudocount:
```
Comment (LeilyR) [inline pygenometracks/utilities.py:264]: why did you change the max to min?

Comment (LeilyR) [reply pygenometracks/utilities.py:264]: this should stay max, i would say. it is enough only if one is below this threshold.

Comment (lldelisle) [reply pygenometracks/utilities.py:264]: No if one is below but not others, the max will be above.

Comment (lldelisle) [reply pygenometracks/utilities.py:264]: Now I am confused... I thought we don't want to apply log to negative values. So if one value is below we don't want to compute, so if the min is below, no?

Comment (LeilyR) [reply pygenometracks/utilities.py:264]: I understood that at the moment we cannot accept any negative value so you are giht and we can change that to min.
